### PR TITLE
Test structure refactor

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

**Comment by mhines-usgs** [comment]
:+1:

**Comment by mhines-usgs** [comment]
I just pulled this and will add to it

File: scratchpad.R [hunk 1]
```diff
@@ -0,0 +1,6 @@
+library(jsonlite)
```
Comment (mhines-usgs) [inline scratchpad.R:1]: did you mean to check this in?

File: scratchpad2.R [hunk 2]
```diff
@@ -0,0 +1,4 @@
+library(repgen)
```
Comment (mhines-usgs) [inline scratchpad2.R:1]: same ? did you mean to check this in?

Comment (thongsav-usgs) [inline scratchpad2.R:1]: Yes to both, they are helpful for reference when not building with RStudio. I can move them if there is a better place for them, but they were supposed to just be helper files during development. I could be convinced to drop them also if @lindsaycarr or @jiwalker-usgs don't like them either.

**Pull request merged**